### PR TITLE
woodwork v0.11.2

### PR DESCRIPTION
**Pull request by tamargrey**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by tamargrey** [comment]
@conda-forge-admin, please rerender

**Review by gsheni — DISMISSED** [review]

File: recipe/meta.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ package:
 
 source:
   url: https://pypi.io/packages/source/w/woodwork/woodwork-{{ version }}.tar.gz
-  sha256: efb47857108d47c929c3159d12aeba8d78ffc6f760fdedaaf70ce977bceae82e
+  sha256: a2f86426f232280ee3cbb237344cc7955b3134ea2500bf42454d71822d652141
 
 build:
   entry_points:
```
Comment (gsheni) [inline recipe/meta.yaml:13]: I think we need to increment build number? @rwedge can you confirm?

>  Bumped the build number (if the version is unchanged)
>  Reset the build number to 0 (if the version changed)

Comment (rwedge) [reply recipe/meta.yaml:13]: We reset the build number to 0 since the version changed

**Review by rwedge — APPROVED** [review]
LGTM

**Pull request merged**